[14.0][MIG] stock_request_analytic

rousseldenis opened the pull request:
Supersedes #1066 #1149

AaronHForgeFlow committed:
[ADD]stock_request_analytic

LoisRForgeFlow committed:
[11.0] stock_request_analytic:
 * clean views.
 * allow to create a stock.request with no analytic account.
 * remove copy attribute in non-stored field.
 * remove non-existing field from compute method.

LoisRForgeFlow committed:
[12.0][MIG] stock_request_analytic

MiquelRForgeFlow committed:
[IMP] Simplify inheritance of states

HviorForgeFlow committed:
[FIX] stock_request_*: add expected_date implicit on tests to avoid errors

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (8 of 8 strings)

Translation: stock-logistics-warehouse-12.0/stock-logistics-warehouse-12.0-stock_request_analytic
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_request_analytic/zh_CN/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (8 of 8 strings)

Translation: stock-logistics-warehouse-12.0/stock-logistics-warehouse-12.0-stock_request_analytic
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-warehouse-12-0/stock-logistics-warehouse-12-0-stock_request_analytic/pt_BR/

joao-p-marques committed:
[IMP] stock_request_analytic: add analytic tags to Stock Requests

TT28488

joao-p-marques committed:
[IMP] stock_request_analytic: black, isort, prettier

joao-p-marques committed:
[MIG] stock_request_analytic: Migration to 13.0

Co-authored-by: Pimolnat Suntian <pimolnats@ecosoft.co.th>

TT28896

oca-travis committed:
[UPD] Update stock_request_analytic.pot

OCA-git-bot committed:
[UPD] README.rst

rousseldenis committed:
[IMP] stock_request_analytic: black, isort, prettier

rousseldenis committed:
[14.0][MIG] stock_request_analytic

patrickrwilson reviewed (approved):
Code Review: LGTM

rousseldenis committed:
[14.0][IMP] stock_request_analytic: Add default analytic account on orders

alan196 reviewed (approved):
👍 Technical and functional review

rousseldenis committed:
[14.0][IMP] stock_request_analytic: Add analytic account to procurement values

If procurement will trigger a purchase and not a move,
there is no mean to transmit it (e.g.: to purchase).

rousseldenis commented:
/ocabot migration stock_request_analytic

rousseldenis commented:
/ocabot merge patch
